README.md

```diff
@@ -147,6 +148,21 @@ return [
          */
         'request_key' => 'exclude',
     ],
+
+    'auto_fieldsets' => [
+
+        /*
+         * If enabled Fractal will automatically add the fieldsets who's
+         * names are present in the `fields` request parameter.
+         */
+        'enabled' => true,
```

Comment:
Isn't this a security issue to enable this by default for everyone who updates to the new version of the package?

Reply:
How could it be a security problem?
I believe `fractal` [removes](https://github.com/thephpleague/fractal/blob/8b9d39b67624db9195c06f9c1ffd0355151eaf62/src/Scope.php#L419C24-L419C39) the fields after the data is transformed and before it is going out.

Reply:
Maybe I'm understanding it wrong (it's been a while since I touched this package), but by enabling this by default, you could just request any field via the URL where previously that would have not been possible.

Reply:
Oh no,
It just to some arrayii stuff after the data is processed already and just remove eveyfields that are not requested.
[This](https://github.com/thephpleague/fractal/blob/8b9d39b67624db9195c06f9c1ffd0355151eaf62/src/Scope.php#L419C24-L419C39) is the method responsible for doing this in phpleague fractal.